eosiojavaabieos/src/main/java/one/block/eosiojavaabieosserializationprovider/AbiEosSerializationProviderImpl.java

```diff
@@ -209,14 +215,43 @@ public String serializeTransaction(String json) throws SerializeTransactionError
     @NotNull
     public String serializeAbi(String json) throws SerializeAbiError {
         try {
-            String abi = getAbiJsonString("abi.abi.json");
+            refreshContext();
+            boolean jsonToBinResult = abiJsonToBin(context, json);
+            if(!jsonToBinResult) {
+                String err = error();
+                String errMsg = String
+                        .format("Unable to pack abi json to bin. %s", err == null ? "" : err);
+                throw new SerializeError(errMsg);
```

Comment:
Please change the "throws" declaration and documentation to match the error type.

Reply:
We catch and wrap these exceptions in the appropriate outer error type further down.